pages/project-version/read/js/index.js

```diff
@@ -20,6 +20,7 @@ start({
     user: `${state.static.user.firstName} ${state.static.user.lastName}`,
     basename: state.static.basename,
     schemaVersion: state.model.project.schemaVersion,
-    establishment: state.static.establishment.name
+    establishment: state.static.establishment.name,
+    changed: state.static.changed
```

Comment:
we might want to pull this out to the top level (as per comments) as it's not going to be a static property - we're going to be manipulating the array client side as and when changes are made - it will need a reducer and some actions in asl-projects

Reply:
done